src/DependencySupport.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class to determine support for AMP plugin features.
+ *
+ * @since 2.1.2
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * DependencySupport class.
+ *
```

Comment:
```suggestion
 *
 * @since 2.1.2
```

Reply:
Done in 260734553a6bbe667fa5d05961c314cc99c37fe3.